consul/metadata.csv

```diff
@@ -23,7 +23,7 @@ consul.runtime.heap_objects,gauge,10,object,,Number of objects allocated on the
 consul.runtime.sys_bytes,gauge,10,byte,,Total size of the virtual address space reserved by the Go runtime,0,consul,
 consul.runtime.malloc_count,gauge,10,object,,Cumulative count of heap objects allocated,0,consul,
 consul.runtime.free_count,gauge,10,object,,Cumulative count of heap objects freed,0,consul,
-consul.runtime.heap_objects,gauge,10,byte,,Bytes of allocated heap objects,0,consul,heap alloc
+consul.runtime.heap_bytes,gauge,10,byte,,Bytes of allocated heap objects,0,consul,heap alloc
```

Comment:
what happened here?

Reply:
heap_objects is a duplicate on line 22, changed this one to better match this description

Reply:
So is this a new metric_name?

Reply:
It doesn't seems to me this metrics exists. See https://www.consul.io/docs/agent/telemetry.html so we should remove it.